robottelo/config/validators.py

```diff
@@ -0,0 +1,219 @@
+from dynaconf import Validator
+
+from robottelo.constants import AZURERM_VALID_REGIONS
+from robottelo.constants import VALID_GCE_ZONES
+
+
+validators = dict(
+    server=[
+        Validator("server.hostname", must_exist=True),
+        Validator("server.ssh_key", must_exist=True)
+        | Validator("server.ssh_password", must_exist=True),
+    ],
+    azurerm=[
+        Validator(
+            "azurerm.client_id",
+            "azurerm.client_secret",
+            "azurerm.subscription_id",
+            "azurerm.tenant_id",
+            "azurerm.azure_region",
+            "azurerm.ssh_pub_key",
+            "azurerm.username",
+            "azurerm.password",
+            "azurerm.azure_subnet",
+            must_exist=True,
+        ),
+        Validator("azurerm.azure_region", is_in=AZURERM_VALID_REGIONS),
+    ],
+    capsule=[Validator("capsule.instance_name", must_exist=True)],
```

Comment:
you'll likely need to add a validator for broker as well

Reply:
Considering that right now validator exceptions are logged and ignored (we must do that until we actually mandate dynaconf on every robottelo user, that is until around phase 3), I'm leaning towards separate, smaller PR that will add all missing validators and remove these that are not needed. Such PR would be submitted after this one is merged.
How does that sound?